examples/using-assertions/test.js

```diff
@@ -0,0 +1,35 @@
+import { Selector } from 'testcafe';
+
+fixture `Use build-in assertions`
+    .page `http://devexpress.github.io/testcafe/example/`;
+
+const triedTestCafeCheckbox = Selector('#tried-test-cafe');
+
+test('Using snapshot API shorthands', async t => {
+    await t
+        .expect(triedTestCafeCheckbox.checked).notOk()
+        .click(triedTestCafeCheckbox)
+        .expect(triedTestCafeCheckbox.checked).ok();
+
+    const slider        = Selector('.ui-slider-handle');
+    const sliderTick    = Selector('.slider-value');
+    const initialOffset = await slider.offsetLeft;
+
+    await t
+        .dragToElement(slider, sliderTick.withText('9'))
+        .expect(slider.offsetLeft).gte(initialOffset);
+});
+
+test('Using DOM node snapshot', async t => {
+    const commentsTextArea = Selector('#comments');
```

Comment:
We've decided to encourage people to not use snapshots there possible. So I guess it'll be better redesigned to use parametrized selector.